README.md

```diff
@@ -1,8 +1,111 @@
-# eventing-auth-manager
-// TODO(user): Add simple overview of use/purpose
+# Eventing Auth Manager
+The eventing-auth-manager is a central component that is deployed on the [Kyma control-plane](https://github.com/kyma-project/control-plane). The component manages applications in the 
+SAP Cloud Identity Services - Identity Authentication (IAS) by creating and deleting them based on creation or deletion of a managed Kyma Runtime.
+
+### Table of contents
+* [Description](#description)
+* [EventingAuth CR](#eventingauth-cr)
+* [eventing-webhook-auth secret](#eventing-webhook-auth-secret)
+* [Name reference between resources](#name-reference-between-resources)
+* [Design decisions](#design-decisions)
+* [Generating the SAP Cloud Identity Services API client](#generating-the-sap-cloud-identity-services-api-client)
+* [Getting Started](#getting-started)
+* [Contributing](#contributing)
+
 
 ## Description
-// TODO(user): An in-depth paragraph about your project and overview of use
+The eventing-auth-manager manages the credentials for IAS applications used by webhooks in the `eventing` component.  
+When a new managed Kyma Runtime is provisioned the component creates a new [OIDC](https://openid.net/connect/) IAS application for this runtime and stores the credentials in a secret on the new runtime.
+
+![eventing-auth-manager-overview](./doc/overview.svg)
```

Comment:
can we use proper names for the secrets `eventing-auth-ias-creds` and `eventing-webhook-auth` in the diagram? 
I think we should rename the target cluster secret name to `eventing-webhook-auth`.

Reply:
I adapter the overview. The renaming is also done in this PR.